src/cmd/single_tablet_test.cc

```diff
@@ -89,7 +89,26 @@ TEST_P(DBSDKTest, CreateFunction) {
                     strlength_sql,
                     int2str_sql
                 });
-    auto result = sr->ExecuteSQL("select cut2(c1), strlength(c1), int2str(c2) from t1;", &status);
+    auto result = sr->ExecuteSQL("show functions", &status);
+    if (cs->IsClusterMode()) {
+        ExpectResultSetStrEq(
+            {{"name", "return_type", "arg_type", "is_aggregate", "file", "offline_file"},
+             {"cut2", "Varchar", "Varchar", "false", "/__w/OpenMLDB/OpenMLDB/build/src/libtest_udf.so",
```

Comment:
`file` filed check may omitted ? it is not constant on other environment

Reply:
just pass `{}` instead

Reply:
use `so_path` defined above